documentation/RELEASE_NOTES.md

```diff
@@ -22,6 +22,21 @@ should also be read to understand what has changed since earlier releases:
 
 ## Changes made on the 7.0 branch since 7.0.8.1
 
+### Changes to msi related to include paths
+
+There are two changes to `msi` included here.
+
+`msi` now treats files included by .template or .substutiions files in a more
+consistent way: for relative paths, it will always look relative to the current
```

Comment:
Can you describe what the previous inconsistency was here in a couple of words, e.g. "on Windows" or whatever the problem was?

Note that msi.md lives adjacent to the msi.cpp file in this repo.

Reply:
I have tried to clarify this somewhat better with an example in each file.